lib/package.json

```diff
@@ -44,6 +44,7 @@
   "scripts": {
     "build": "npm run clean && webpack && tsc -p tsconfig-esm.json && node prepareNodeESM.js && ../scripts/add-module-types.sh esm && ../scripts/fill-version.sh",
     "clean": "rm -rf {build,dist}",
+    "updateLink": "npm run build && sudo npm rm @opentdf/client -g && sudo npm link",
```

Comment:
do we rly need it ?

Reply:
how is this used?  can you add to README